src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Reissuance/ReissuanceConsent/__tests__/HealthCertificateReissuanceConsentViewModelTests.swift

```diff
@@ -50,7 +50,7 @@ class HealthCertificateReissuanceConsentViewModelTests: CWATestCase {
 		let certificate = HealthCertificate.mock()
 		let viewModel = HealthCertificateReissuanceConsentViewModel(
 			cclService: FakeCCLService(),
-			certificates: [certificate],
+			certificates: [DCCReissuanceCertificateContainer.fake()],
```

Comment:
```suggestion
			certificates: [.fake()],
```
Same for the ones below, type should be inferred from context